config/initializers/blacklight_initializer.rb

```diff
@@ -24,39 +24,57 @@ def export_as_openurl_ctx_kev(format = nil)
     id = to_marc.find{|field| field.tag == '001'}
```

Comment:
The idea is that we will move the entire export_as_openurl_ctx_kev function logic (and any attendant helper code) to orangeindex and just inject an OpenURL into the solr document. Correct @tampakis.